.evergreen/csfle/activate-kmstlsvenv.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
```

Comment:
Aren't shebangs meaningless (ignored) for scripts that are intended to be sourced? Eg changing this to `#!/usr/bin/env sh` would have no effect.

Reply:
Gotcha thanks for explaining that.